pkg/controller/lifecycle/add.go

```diff
@@ -62,25 +63,25 @@ func configMapToAllExtensionMapper(ctx context.Context, log logr.Logger, reader
 	extensionsFound := []string{}
 	for _, extension := range extensionList.Items {
 		if extension.Spec.Type == consts.ExtensionType {
-			isOk := false
-			// Assume, there is only one condition and it is is Ok "True", then add this extension to requests for reconciliation
-			for _, condition := range extension.Status.Conditions {
-				isOk = (condition.Status == "True")
-				break
-			}
-			if isOk {
-				requests = append(requests, reconcile.Request{
-					NamespacedName: types.NamespacedName{
-						Namespace: extension.Namespace,
-						Name:      extension.Name,
-					},
-				})
-				extensionsFound = append(extensionsFound, fmt.Sprintf("%s/%s", extension.Namespace, extension.Name))
+
+			// Skip extensions in "processing" state to not race with original extension controller
+			// https://github.com/gardener/gardener/blob/5bf28f8ff7ecf3e2ffe21224f0cb6ee30daf9997/extensions/pkg/controller/status.go#L115
+			// Optionally take every that previously succeeded: if !(extension.Status.LastOperation.State == gardencorev1beta1.LastOperationStateSucceeded) {
```

Comment:
Every what? Extension? I don't get it.

Reply:
I removed all of it - it was test (now dead) code I wanted to try but above works fine.